### PR TITLE
Schemes should be registered with IANA

### PR DESCRIPTION
**Pull request by cvrebert**
At least, I'm assuming that's what the intention was for this XXX'd sentence.

### CODE REVIEW

**Comment by annevk** [comment]
It's not entirely clear yet whether that is the way to go. Since the IETF defines URLs in a different way.

**Comment by domenic** [comment]
I would just remove that sentence... or at least make it much clearer that it's an authoring conformance requirement of some sort.

**Comment by masinter** [comment]
bcp35 was just updated (RFC 7595O to make it easier to register schemes,
to prevent collisions and inform everyone what the scheme means.

People producing or interpreting url schemes should follow the registered definition.

And if you’re inventing a new scheme you should follow the guidelines
for URL schemes.

All of these are recommendations for good practice.

The URL standard is a logical place to inform people about this, not as
a conformance requirement but as just some “good citizen” advice.




On 7/13/15, 7:59 AM, "Domenic Denicola" <notifications@github.com<mailto:notifications@github.com>> wrote:


I would just remove that sentence... or at least make it much clearer that it's an authoring conformance requirement of some sort.

**Comment by cvrebert** [comment]
Pushed a new commit with revised phrasing:
"Authors should ensure that all schemes that they use are registered in the IANA URI Schemes registry."

File: url.bs [hunk 1]
```diff
@@ -848,8 +848,10 @@ followed by "<code>?</code>" and a <a lt="URL query">query</a>.
 
 <p>A <a lt="URL scheme">scheme</a> must be one <a>ASCII alpha</a>, followed by zero or
 more of <a>ASCII alphanumeric</a>, "<code>+</code>", "<code>-</code>", and
-"<code>.</code>". A <a lt="URL scheme">scheme</a> must be registered
-<span class=XXX>...</span>.
+"<code>.</code>". Authors should ensure that all <a lt="URL scheme">schemes</a> that they
+use are registered in the IANA URI Schemes registry.
```
Comment (annevk) [inline url.bs]: Please say "Developers" instead of "Authors" and please add [sic] after URI.

Comment (cvrebert) [inline url.bs]: Done.

**Pull request merged**